core/templates/dev/head/admin/Admin.js

```diff
@@ -30,6 +30,25 @@ oppia.controller('Admin', ['$scope', '$http', function($scope, $http) {
 
   $scope.currentTab = $scope.TAB_ACTIVITIES;
 
+  $scope.$watch(function() {
+    return window.location.hash;
+  }, function(newHash) {
+    switch (newHash) {
+      case '#jobs':
+        $scope.currentTab = $scope.TAB_JOBS;
+        break;
+      case '#config':
+        $scope.currentTab = $scope.TAB_CONFIG;
+        break;
+      case '#misc':
+        $scope.currentTab = $scope.TAB_MISC;
+        break;
+      case '#activities':
+        $scope.currentTab = $scope.TAB_ACTIVITIES;
+        break;
```

Comment:
I tried that initially, the e2e tests were failing because of that so had to change it.